components/core/src/fs.rs

```diff
@@ -201,13 +201,39 @@ pub fn find_command(command: &str) -> Option<PathBuf> {
                 if candidate.is_file() {
                     return Some(candidate);
                 }
+                else {
+                    match find_command_with_pathext(&candidate) {
```

Comment:
Thanks for the feedback!

I think that would only work for the first path in the path variable checked.  The second part of the match `None => {}` is what lets things proceed on to the next path to check.

I'm going to write some tests to validate the behavior.